context/app/static/js/components/detailPage/Attribution/Attribution.jsx

```diff
@@ -24,10 +22,4 @@ function Attribution({ group_name, created_by_user_displayname, created_by_user_
   );
 }
 
-Attribution.propTypes = {
```

Comment:
Maybe we should keep the proptypes but comment them out instead of dropping them altogether - this would aid us in future TypeScript conversions.